distribution/feature-packs/server-feature-pack/src/main/resources/licenses/rh-sso/licenses.xml

```diff
@@ -521,22 +521,22 @@
     <dependency>
       <groupId>com.webauthn4j</groupId>
       <artifactId>webauthnj4-core</artifactId>
-      <version>0.9.14.RELEASE</version>
+      <version>0.10.2.RELEASE</version>
```

Comment:
This should should be updated after we have a internal build with redhat suffix.

Reply:
@martin-kanis Is it something , which needs to be done in this PR or could it be done as a follow-up PR? Whatever of those options works for me.

In any case, we should not merge this PR until internal build is successfully finished. @drichtarik is already working on it. @drichtarik Could you please comment here what would be the correct version to be used here once you finished the internal build?

Reply:
It's cleaner to do it as a part of this PR. However if this blocks something it can be done as follow up.